components/http_foundation/introduction.rst

```diff
@@ -165,8 +161,16 @@ argument::
     $request->query->get('foo[bar]');
     // returns null
 
-    $request->query->get('foo[bar]', null, true);
-    // returns 'bar'
+.. versionadded:: 3.0
+    It is no longer possible to get the value of "original" parameter ``foo[bar]``.
+    Prior, you were able to do this via the third argument of 
+    :method:`Symfony\\Component\\HttpFoundation\\Request::get`:
+
+    .. code-block:: php
+    
+        $request->query->get('foo[bar]', null, true);
+        // returns 'bar'
```

Comment:
I think we shouldn't simply talk about this. People will notice they have to change their code when getting deprecation notices with Symfony 2.8.